src/tekton/pipelinecontent.ts

```diff
@@ -22,7 +22,7 @@ export interface NameType {
 
 export interface ItemValue {
   key?: string;
-  value?: string;
+  path?: string;
```

Comment:
In this case you need to change type name `ItemValue` -> `ItemPath`, or keep `value`

In other hand we already have absolutely the same type `ItemValue`